Failing Cleaners Tests

I just started the open source game, so forgive me if I'm doing it wrong.  

```
$ python manage.py test
...
tests/test_cleaners.py::TestCleaners::test_unknown_resident_weapon_returned FAILED   
                                                                                     
===================================== FAILURES ======================================
________________ TestCleaners.test_unknown_resident_weapon_returned _________________
                                                                                     
self = <tests.test_cleaners.TestCleaners instance at 0x7f1847217758>                 
                                                                                     
    def test_unknown_resident_weapon_returned(self):                                 
        ''' Cleaning an unknown weapon returns the same value passed.                
            '''                                                                      
        weapon = "Suspect - Lewd Nebulosity "                                        
>       cleaned = Cleaners.resident_weapon_used(weapon)                              
E       TypeError: unbound method resident_weapon_used() must be called with Cleaners instance as first argument (got str instance instead)                               
                                                                                     
tests/test_cleaners.py:11: TypeError                                                 
!!!!!!!!!!!!!!!!!!!!!! Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!
======================== 1 failed, 5 passed in 1.63 seconds =========================
```

It seems like the Cleaners functions, like `residen_weapon_used()` should be performed on an instance, not the class.  

I would think this could be fixed by adding `@staticmethod` to each of the Cleaners functions.  Or by creating a Cleaners instance in TestCleaners: 

```
class TestCleaners:
    cleaner = Cleaners()

    def test_unknown_resident_weapon_returned(self):
        ''' Cleaning an unknown weapon returns the same value passed.
        '''
        weapon = "Suspect - Lewd Nebulosity "
        cleaned = cleaner.resident_weapon_used(weapon)
        assert cleaned == weapon
...
```

--- a/comport/data/cleaners.py
+++ b/comport/data/cleaners.py
@@ -55,20 +55,20 @@ CAPITALIZE_IGNORE_KEYS_LIST = [
     "officerYearsOfService"
 ]
 
-class Cleaners:
+class Cleaners(object):
 
-    def resident_weapon_used(val):
+    def resident_weapon_used(self, val):
         if val in RESIDENT_WEAPONS_LOOKUP:
             return RESIDENT_WEAPONS_LOOKUP[val]
         else:
             return val
 
-    def officer_force_type(text):
+    def officer_force_type(self, text):
         if text is not None and text in OFFICER_FORCE_TYPE_LOOKUP:
             return OFFICER_FORCE_TYPE_LOOKUP[text]
         return text
 
-    def race(text):
+    def race(self, text):
         if not text:
             return None
 
@@ -79,7 +79,7 @@ class Cleaners:
         else:
             return titlecase(text)
 
-    def sex(text):
+    def sex(self, text):
         if not text:
             return None
 
@@ -90,7 +90,7 @@ class Cleaners:
         else:
             return titlecase(text)
 
-    def number_to_string(value):
+    def number_to_string(self, value):
         ''' If it's a number, turn the passed value into a string, everything else is returned as-is.
         '''
         type_value = type(value)
@@ -98,7 +98,7 @@ class Cleaners:
             return str(value)
         return value
 
-    def string_to_integer(value):
+    def string_to_integer(self, value):
         ''' Strings and floats are turned to integers, integers are returned as-is, everything else is None.
         '''
         if type(value) == str:
@@ -117,7 +117,7 @@ class Cleaners:
             return int(value)
         return None
 
-    def capitalize(value):
+    def capitalize(self, value):
         def abbreviations(word, **kwargs):
             if word.upper() in CAPITALIZE_LIST:
                 return word.upper()
@@ -126,11 +126,11 @@ class Cleaners:
             return value
         return titlecase(value.strip().lower(), callback=abbreviations)
 
-    def capitalize_incident(incident):
+    def capitalize_incident(self, incident):
         for key in list(incident.keys()):
             if key in CAPITALIZE_IGNORE_KEYS_LIST:
                 continue
 
-            incident[key] = Cleaners.capitalize(incident[key])
+            incident[key] = self.capitalize(incident[key])
 
         return incident

--- a/comport/data/views.py
+++ b/comport/data/views.py
@@ -38,24 +38,25 @@ def use_of_force():
     j = request.json
     added_rows = 0
     updated_rows = 0
+    cleaner = Cleaners()
 
     for incident in j['data']:
 
         # capitalize the location
-        incident["division"] = Cleaners.capitalize(incident["division"])
-        incident["precinct"] = Cleaners.capitalize(incident["precinct"])
-        incident["shift"] = Cleaners.capitalize(incident["shift"])
-        incident["beat"] = Cleaners.capitalize(incident["beat"])
+        incident["division"] = cleaner.capitalize(incident["division"])
+        incident["precinct"] = cleaner.capitalize(incident["precinct"])
+        incident["shift"] = cleaner.capitalize(incident["shift"])
+        incident["beat"] = cleaner.capitalize(incident["beat"])
         # clean force type, race, gender
-        incident["officerForceType"] = Cleaners.officer_force_type(incident["officerForceType"])
-        incident["residentRace"] = Cleaners.race(incident["residentRace"])
-        incident["residentSex"] = Cleaners.sex(incident["residentSex"])
-        incident["officerRace"] = Cleaners.race(incident["officerRace"])
-        incident["officerSex"] = Cleaners.sex(incident["officerSex"])
+        incident["officerForceType"] = cleaner.officer_force_type(incident["officerForceType"])
+        incident["residentRace"] = cleaner.race(incident["residentRace"])
+        incident["residentSex"] = cleaner.sex(incident["residentSex"])
+        incident["officerRace"] = cleaner.race(incident["officerRace"])
+        incident["officerSex"] = cleaner.sex(incident["officerSex"])
         # make sure values that might've been sent as integers are strings
-        incident["residentAge"] = Cleaners.number_to_string(incident["residentAge"])
-        incident["officerAge"] = Cleaners.number_to_string(incident["officerAge"])
-        incident["officerYearsOfService"] = Cleaners.number_to_string(incident["officerYearsOfService"])
+        incident["residentAge"] = cleaner.number_to_string(incident["residentAge"])
+        incident["officerAge"] = cleaner.number_to_string(incident["officerAge"])
+        incident["officerYearsOfService"] = cleaner.number_to_string(incident["officerYearsOfService"])
 
         found_incident = UseOfForceIncident.query.filter_by(
             opaque_id=incident["opaqueId"],
@@ -148,25 +149,26 @@ def officer_involved_shooting():
     request_json = request.json
     added_rows = 0
     updated_rows = 0
+    cleaner = Cleaners()
 
     for incident in request_json['data']:
 
         # capitalize the location
-        incident["division"] = Cleaners.capitalize(incident["division"])
-        incident["precinct"] = Cleaners.capitalize(incident["precinct"])
-        incident["shift"] = Cleaners.capitalize(incident["shift"])
-        incident["beat"] = Cleaners.capitalize(incident["beat"])
+        incident["division"] = cleaner.capitalize(incident["division"])
+        incident["precinct"] = cleaner.capitalize(incident["precinct"])
+        incident["shift"] = cleaner.capitalize(incident["shift"])
+        incident["beat"] = cleaner.capitalize(incident["beat"])
         # clean weapon, race, gender
-        incident["residentWeaponUsed"] = Cleaners.resident_weapon_used(incident["residentWeaponUsed"])
-        incident["residentSex"] = Cleaners.sex(incident["residentSex"])
-        incident["residentRace"] = Cleaners.race(incident["residentRace"])
-        incident["officerSex"] = Cleaners.sex(incident["officerSex"])
-        incident["officerRace"] = Cleaners.race(incident["officerRace"])
+        incident["residentWeaponUsed"] = cleaner.resident_weapon_used(incident["residentWeaponUsed"])
+        incident["residentSex"] = cleaner.sex(incident["residentSex"])
+        incident["residentRace"] = cleaner.race(incident["residentRace"])
+        incident["officerSex"] = cleaner.sex(incident["officerSex"])
+        incident["officerRace"] = cleaner.race(incident["officerRace"])
         # make sure values that might've been sent as integers are strings
-        incident["residentAge"] = Cleaners.number_to_string(incident["residentAge"])
-        incident["officerAge"] = Cleaners.number_to_string(incident["officerAge"])
+        incident["residentAge"] = cleaner.number_to_string(incident["residentAge"])
+        incident["officerAge"] = cleaner.number_to_string(incident["officerAge"])
         # and values that might've been sent as strings are integers
-        incident["officerYearsOfService"] = Cleaners.string_to_integer(incident["officerYearsOfService"])
+        incident["officerYearsOfService"] = cleaner.string_to_integer(incident["officerYearsOfService"])
 
         found_incident = OfficerInvolvedShooting.query.filter_by(
             opaque_id=incident["opaqueId"],
@@ -244,19 +246,20 @@ def complaints():
     j = request.json
     added_rows = 0
     updated_rows = 0
+    cleaner = Cleaners()
 
     for incident in j['data']:
         # make sure values that might've been sent as integers are strings
-        incident["residentAge"] = Cleaners.number_to_string(incident["residentAge"])
-        incident["officerAge"] = Cleaners.number_to_string(incident["officerAge"])
-        incident["officerYearsOfService"] = Cleaners.number_to_string(incident["officerYearsOfService"])
+        incident["residentAge"] = cleaner.number_to_string(incident["residentAge"])
+        incident["officerAge"] = cleaner.number_to_string(incident["officerAge"])
+        incident["officerYearsOfService"] = cleaner.number_to_string(incident["officerYearsOfService"])
         # capitalize all the fields in the incident
-        incident = Cleaners.capitalize_incident(incident)
+        incident = cleaner.capitalize_incident(incident)
         # clean sex & race
-        incident["residentSex"] = Cleaners.sex(incident["residentSex"])
-        incident["residentRace"] = Cleaners.race(incident["residentRace"])
-        incident["officerSex"] = Cleaners.sex(incident["officerSex"])
-        incident["officerRace"] = Cleaners.race(incident["officerRace"])
+        incident["residentSex"] = cleaner.sex(incident["residentSex"])
+        incident["residentRace"] = cleaner.race(incident["residentRace"])
+        incident["officerSex"] = cleaner.sex(incident["officerSex"])
+        incident["officerRace"] = cleaner.race(incident["officerRace"])
 
         found_incident = CitizenComplaint.query.filter_by(
             opaque_id=incident["opaqueId"],

--- a/tests/test_cleaners.py
+++ b/tests/test_cleaners.py
@@ -7,78 +7,88 @@ class TestCleaners:
     def test_unknown_resident_weapon_returned(self):
         ''' Cleaning an unknown weapon returns the same value passed.
         '''
+        cleaner = Cleaners()
         weapon = "Suspect - Lewd Nebulosity "
-        cleaned = Cleaners.resident_weapon_used(weapon)
+        cleaned = cleaner.resident_weapon_used(weapon)
         assert cleaned == weapon
 
     def test_known_resident_weapons_cleaned(self):
         ''' Cleaning known weapons returns the expected value.
         '''
+        cleaner = Cleaners()
         for weapon in RESIDENT_WEAPONS_LOOKUP:
-            check_weapon = Cleaners.resident_weapon_used(weapon)
+            check_weapon = cleaner.resident_weapon_used(weapon)
             assert check_weapon == RESIDENT_WEAPONS_LOOKUP[weapon]
 
     def test_unknown_officer_force_type_returned(self):
         ''' Cleaning an unknown weapon returns the same value passed.
         '''
+        cleaner = Cleaners()
         force_type = "Bugbear Squeeze"
-        cleaned = Cleaners.officer_force_type(force_type)
+        cleaned = cleaner.officer_force_type(force_type)
         assert cleaned == force_type
 
     def test_known_officer_force_types_cleaned(self):
         ''' Cleaning known weapons returns the expected value.
         '''
+        cleaner = Cleaners()
         for force_type in OFFICER_FORCE_TYPE_LOOKUP:
-            check_force_type = Cleaners.officer_force_type(force_type)
+            check_force_type = cleaner.officer_force_type(force_type)
             assert check_force_type == OFFICER_FORCE_TYPE_LOOKUP[force_type]
 
     def test_unknown_race_returned(self):
         ''' Cleaning an unknown race returns the same value passed.
         '''
+        cleaner = Cleaners()
         race = "Comet"
-        cleaned = Cleaners.race(race)
+        cleaned = cleaner.race(race)
         assert cleaned == race
 
     def test_known_races_cleaned(self):
         ''' Cleaning known races returns the expected value.
         '''
+        cleaner = Cleaners()
         for race in RACE_LOOKUP:
-            check_race = Cleaners.race(race)
+            check_race = cleaner.race(race)
             assert check_race == RACE_LOOKUP[race]
 
     def test_unknown_gender_returned(self):
         ''' Cleaning an unknown gender returns the same value passed.
         '''
+        cleaner = Cleaners()
         gender = "Golf"
-        cleaned = Cleaners.sex(gender)
+        cleaned = cleaner.sex(gender)
         assert cleaned == gender
 
     def test_known_genders_cleaned(self):
         ''' Cleaning known genders returns the expected value.
         '''
+        cleaner = Cleaners()
         for gender in GENDER_LOOKUP:
-            check_gender = Cleaners.sex(gender)
+            check_gender = cleaner.sex(gender)
             assert check_gender == GENDER_LOOKUP[gender]
 
     def test_capitalization(self):
         ''' A phrase is title-cased with expected exceptions.
         '''
+        cleaner = Cleaners()
         in_sentence = "I thought I would sail about a little and see the watery part of the world"
         titlecased_sentence = titlecase(in_sentence)
         send = " ".join(i.lower() + " " + j.lower() for i, j in zip(in_sentence.split(" "), CAPITALIZE_LIST))
         check = " ".join(i + " " + j for i, j in zip(titlecased_sentence.split(" "), CAPITALIZE_LIST))
-        result = Cleaners.capitalize(send)
+        result = cleaner.capitalize(send)
         assert check == result
 
     def test_captilization_handles_non_strings(self):
         ''' Non-strings passed to capitalize aren't altered.
         '''
+        cleaner = Cleaners()
         send_int = 23
         send_float = 32.01
         send_list = ["I", "thought", "I", "would", "sail"]
-        result_int = Cleaners.capitalize(send_int)
-        result_float = Cleaners.capitalize(send_float)
-        result_list = Cleaners.capitalize(send_list)
+        result_int = cleaner.capitalize(send_int)
+        result_float = cleaner.capitalize(send_float)
+        result_list = cleaner.capitalize(send_list)
         assert send_int == result_int
         assert send_float == result_float
         assert send_list == result_list
@@ -86,6 +96,7 @@ class TestCleaners:
     def test_incident_capitalization(self):
         ''' Incident descriptions are title-cased with expected exceptions.
         '''
+        cleaner = Cleaners()
         in_sentence = "It is a way I have of driving off the spleen and regulating the circulation"
         titlecased_sentence = titlecase(in_sentence)
         send = {}
@@ -94,7 +105,7 @@ class TestCleaners:
         for key in CAPITALIZE_IGNORE_KEYS_LIST:
             send[key] = in_sentence
 
-        result = Cleaners.capitalize_incident(send)
+        result = cleaner.capitalize_incident(send)
 
         # values of keys not in the list should be titlecased
         assert result["damp"] == titlecased_sentence
@@ -106,20 +117,22 @@ class TestCleaners:
     def test_number_to_string(self):
         ''' Numbers are turned into strings.
         '''
+        cleaner = Cleaners()
         in_int = 85
         in_float = 82.12
         in_string = "big frame, small spirit!"
         in_list = ["hands", "by", "the", "halyards"]
         in_none = None
-        assert Cleaners.number_to_string(in_int) == str(in_int)
-        assert Cleaners.number_to_string(in_float) == str(in_float)
-        assert Cleaners.number_to_string(in_string) == in_string
-        assert Cleaners.number_to_string(in_list) == in_list
-        assert Cleaners.number_to_string(in_none) is None
+        assert cleaner.number_to_string(in_int) == str(in_int)
+        assert cleaner.number_to_string(in_float) == str(in_float)
+        assert cleaner.number_to_string(in_string) == in_string
+        assert cleaner.number_to_string(in_list) == in_list
+        assert cleaner.number_to_string(in_none) is None
 
     def test_string_to_integer(self):
         ''' Strings are turned into integers.
         '''
+        cleaner = Cleaners()
         in_string_success_int = '85'
         in_string_success_float = '33.34'
         in_int = 85
@@ -127,10 +140,10 @@ class TestCleaners:
         in_string_fail = 'whaleman'
         in_list = ["hands", "by", "the", "halyards"]
         in_none = None
-        assert Cleaners.string_to_integer(in_string_success_int) == int(in_string_success_int)
-        assert Cleaners.string_to_integer(in_string_success_float) == int(float(in_string_success_float))
-        assert Cleaners.string_to_integer(in_int) == in_int
-        assert Cleaners.string_to_integer(in_float) == int(in_float)
-        assert Cleaners.string_to_integer(in_string_fail) is None
-        assert Cleaners.string_to_integer(in_list) is None
-        assert Cleaners.string_to_integer(in_none) is None
+        assert cleaner.string_to_integer(in_string_success_int) == int(in_string_success_int)
+        assert cleaner.string_to_integer(in_string_success_float) == int(float(in_string_success_float))
+        assert cleaner.string_to_integer(in_int) == in_int
+        assert cleaner.string_to_integer(in_float) == int(in_float)
+        assert cleaner.string_to_integer(in_string_fail) is None
+        assert cleaner.string_to_integer(in_list) is None
+        assert cleaner.string_to_integer(in_none) is None

--- a/tests/test_extractor_api.py
+++ b/tests/test_extractor_api.py
@@ -70,7 +70,8 @@ class TestHeartbeat:
         assert response.json_body['added'] == complaint_count
 
         # check the complaint incident in the database against the data that was sent
-        sent_complaint = Cleaners.capitalize_incident(complaint_data[0])
+        cleaner = Cleaners()
+        sent_complaint = cleaner.capitalize_incident(complaint_data[0])
         check_complaint = CitizenComplaint.query.filter_by(opaque_id=sent_complaint['opaqueId']).first()
         assert check_complaint.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_complaint['occuredDate']
         assert check_complaint.division == sent_complaint['division']
@@ -82,14 +83,14 @@ class TestHeartbeat:
         assert check_complaint.source == sent_complaint['source']
         assert check_complaint.allegation_type == sent_complaint['allegationType']
         assert check_complaint.allegation == sent_complaint['allegation']
-        assert check_complaint.resident_race == Cleaners.race(sent_complaint['residentRace'])
-        assert check_complaint.resident_sex == Cleaners.sex(sent_complaint['residentSex'])
-        assert check_complaint.resident_age == Cleaners.number_to_string(sent_complaint['residentAge'])
+        assert check_complaint.resident_race == cleaner.race(sent_complaint['residentRace'])
+        assert check_complaint.resident_sex == cleaner.sex(sent_complaint['residentSex'])
+        assert check_complaint.resident_age == cleaner.number_to_string(sent_complaint['residentAge'])
         assert check_complaint.officer_identifier == sent_complaint['officerIdentifier']
-        assert check_complaint.officer_race == Cleaners.race(sent_complaint['officerRace'])
-        assert check_complaint.officer_sex == Cleaners.sex(sent_complaint['officerSex'])
-        assert check_complaint.officer_age == Cleaners.number_to_string(sent_complaint['officerAge'])
-        assert check_complaint.officer_years_of_service == Cleaners.number_to_string(sent_complaint['officerYearsOfService'])
+        assert check_complaint.officer_race == cleaner.race(sent_complaint['officerRace'])
+        assert check_complaint.officer_sex == cleaner.sex(sent_complaint['officerSex'])
+        assert check_complaint.officer_age == cleaner.number_to_string(sent_complaint['officerAge'])
+        assert check_complaint.officer_years_of_service == cleaner.number_to_string(sent_complaint['officerYearsOfService'])
 
     def test_post_mistyped_complaint_data(self, testapp):
         ''' New complaint data from the extractor with wrongly typed data is processed as expected.
@@ -120,7 +121,8 @@ class TestHeartbeat:
         assert response.json_body['added'] == complaint_count
 
         # check the complaint incident in the database against the data that was sent
-        sent_complaint = Cleaners.capitalize_incident(complaint_data[0])
+        cleaner = Cleaners()
+        sent_complaint = cleaner.capitalize_incident(complaint_data[0])
         check_complaint = CitizenComplaint.query.filter_by(opaque_id=sent_complaint['opaqueId']).first()
         assert check_complaint.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_complaint['occuredDate']
         assert check_complaint.division == sent_complaint['division']
@@ -132,14 +134,14 @@ class TestHeartbeat:
         assert check_complaint.source == sent_complaint['source']
         assert check_complaint.allegation_type == sent_complaint['allegationType']
         assert check_complaint.allegation == sent_complaint['allegation']
-        assert check_complaint.resident_race == Cleaners.race(sent_complaint['residentRace'])
-        assert check_complaint.resident_sex == Cleaners.sex(sent_complaint['residentSex'])
-        assert check_complaint.resident_age == Cleaners.number_to_string(sent_complaint['residentAge'])
+        assert check_complaint.resident_race == cleaner.race(sent_complaint['residentRace'])
+        assert check_complaint.resident_sex == cleaner.sex(sent_complaint['residentSex'])
+        assert check_complaint.resident_age == cleaner.number_to_string(sent_complaint['residentAge'])
         assert check_complaint.officer_identifier == sent_complaint['officerIdentifier']
-        assert check_complaint.officer_race == Cleaners.race(sent_complaint['officerRace'])
-        assert check_complaint.officer_sex == Cleaners.sex(sent_complaint['officerSex'])
-        assert check_complaint.officer_age == Cleaners.number_to_string(sent_complaint['officerAge'])
-        assert check_complaint.officer_years_of_service == Cleaners.number_to_string(sent_complaint['officerYearsOfService'])
+        assert check_complaint.officer_race == cleaner.race(sent_complaint['officerRace'])
+        assert check_complaint.officer_sex == cleaner.sex(sent_complaint['officerSex'])
+        assert check_complaint.officer_age == cleaner.number_to_string(sent_complaint['officerAge'])
+        assert check_complaint.officer_years_of_service == cleaner.number_to_string(sent_complaint['officerYearsOfService'])
 
     def test_update_complaint_data(self, testapp):
         ''' Updated complaint data from the extractor is processed as expected.
@@ -186,7 +188,8 @@ class TestHeartbeat:
         assert len(all_complaints) == 1
 
         # check the complaint incident in the database against the updated data that was sent
-        sent_complaint = Cleaners.capitalize_incident(updated_complaint_data[0])
+        cleaner = Cleaners()
+        sent_complaint = cleaner.capitalize_incident(updated_complaint_data[0])
         check_complaint = CitizenComplaint.query.filter_by(opaque_id=sent_complaint['opaqueId']).first()
         assert check_complaint.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_complaint['occuredDate']
         assert check_complaint.division == sent_complaint['division']
@@ -198,14 +201,14 @@ class TestHeartbeat:
         assert check_complaint.source == sent_complaint['source']
         assert check_complaint.allegation_type == sent_complaint['allegationType']
         assert check_complaint.allegation == sent_complaint['allegation']
-        assert check_complaint.resident_race == Cleaners.race(sent_complaint['residentRace'])
-        assert check_complaint.resident_sex == Cleaners.sex(sent_complaint['residentSex'])
-        assert check_complaint.resident_age == Cleaners.number_to_string(sent_complaint['residentAge'])
+        assert check_complaint.resident_race == cleaner.race(sent_complaint['residentRace'])
+        assert check_complaint.resident_sex == cleaner.sex(sent_complaint['residentSex'])
+        assert check_complaint.resident_age == cleaner.number_to_string(sent_complaint['residentAge'])
         assert check_complaint.officer_identifier == sent_complaint['officerIdentifier']
-        assert check_complaint.officer_race == Cleaners.race(sent_complaint['officerRace'])
-        assert check_complaint.officer_sex == Cleaners.sex(sent_complaint['officerSex'])
-        assert check_complaint.officer_age == Cleaners.number_to_string(sent_complaint['officerAge'])
-        assert check_complaint.officer_years_of_service == Cleaners.number_to_string(sent_complaint['officerYearsOfService'])
+        assert check_complaint.officer_race == cleaner.race(sent_complaint['officerRace'])
+        assert check_complaint.officer_sex == cleaner.sex(sent_complaint['officerSex'])
+        assert check_complaint.officer_age == cleaner.number_to_string(sent_complaint['officerAge'])
+        assert check_complaint.officer_years_of_service == cleaner.number_to_string(sent_complaint['officerYearsOfService'])
 
     def test_skip_multiple_complaint_data(self, testapp):
         ''' Multiple complaint data from the extractor is skipped.
@@ -308,15 +311,16 @@ class TestHeartbeat:
         assert response.json_body['added'] == uof_count
 
         # check the uof incident in the database against the data that was sent
+        cleaner = Cleaners()
         sent_uof = uof_data[0]
         check_uof = UseOfForceIncident.query.filter_by(opaque_id=sent_uof['opaqueId']).first()
         assert check_uof.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_uof['occuredDate']
-        assert check_uof.division == Cleaners.capitalize(sent_uof['division'])
-        assert check_uof.precinct == Cleaners.capitalize(sent_uof['precinct'])
-        assert check_uof.shift == Cleaners.capitalize(sent_uof['shift'])
-        assert check_uof.beat == Cleaners.capitalize(sent_uof['beat'])
+        assert check_uof.division == cleaner.capitalize(sent_uof['division'])
+        assert check_uof.precinct == cleaner.capitalize(sent_uof['precinct'])
+        assert check_uof.shift == cleaner.capitalize(sent_uof['shift'])
+        assert check_uof.beat == cleaner.capitalize(sent_uof['beat'])
         assert check_uof.disposition == sent_uof['disposition']
-        assert check_uof.officer_force_type == Cleaners.officer_force_type(sent_uof['officerForceType'])
+        assert check_uof.officer_force_type == cleaner.officer_force_type(sent_uof['officerForceType'])
         assert check_uof.use_of_force_reason == sent_uof['useOfForceReason']
         assert check_uof.service_type == sent_uof['serviceType']
         assert check_uof.arrest_made == sent_uof['arrestMade']
@@ -326,15 +330,15 @@ class TestHeartbeat:
         assert check_uof.resident_hospitalized == sent_uof['residentHospitalized']
         assert check_uof.officer_injured == sent_uof['officerInjured']
         assert check_uof.officer_hospitalized == sent_uof['officerHospitalized']
-        assert check_uof.resident_race == Cleaners.race(sent_uof['residentRace'])
-        assert check_uof.resident_sex == Cleaners.sex(sent_uof['residentSex'])
-        assert check_uof.resident_age == Cleaners.number_to_string(sent_uof['residentAge'])
+        assert check_uof.resident_race == cleaner.race(sent_uof['residentRace'])
+        assert check_uof.resident_sex == cleaner.sex(sent_uof['residentSex'])
+        assert check_uof.resident_age == cleaner.number_to_string(sent_uof['residentAge'])
         assert check_uof.resident_condition == sent_uof['residentCondition']
         assert check_uof.officer_identifier == sent_uof['officerIdentifier']
-        assert check_uof.officer_race == Cleaners.race(sent_uof['officerRace'])
-        assert check_uof.officer_sex == Cleaners.sex(sent_uof['officerSex'])
-        assert check_uof.officer_age == Cleaners.number_to_string(sent_uof['officerAge'])
-        assert check_uof.officer_years_of_service == Cleaners.number_to_string(sent_uof['officerYearsOfService'])
+        assert check_uof.officer_race == cleaner.race(sent_uof['officerRace'])
+        assert check_uof.officer_sex == cleaner.sex(sent_uof['officerSex'])
+        assert check_uof.officer_age == cleaner.number_to_string(sent_uof['officerAge'])
+        assert check_uof.officer_years_of_service == cleaner.number_to_string(sent_uof['officerYearsOfService'])
         assert check_uof.officer_condition == sent_uof['officerCondition']
 
     def test_post_mistyped_uof_data(self, testapp):
@@ -366,15 +370,16 @@ class TestHeartbeat:
         assert response.json_body['added'] == uof_count
 
         # check the uof incident in the database against the data that was sent
+        cleaner = Cleaners()
         sent_uof = uof_data[0]
         check_uof = UseOfForceIncident.query.filter_by(opaque_id=sent_uof['opaqueId']).first()
         assert check_uof.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_uof['occuredDate']
-        assert check_uof.division == Cleaners.capitalize(sent_uof['division'])
-        assert check_uof.precinct == Cleaners.capitalize(sent_uof['precinct'])
-        assert check_uof.shift == Cleaners.capitalize(sent_uof['shift'])
-        assert check_uof.beat == Cleaners.capitalize(sent_uof['beat'])
+        assert check_uof.division == cleaner.capitalize(sent_uof['division'])
+        assert check_uof.precinct == cleaner.capitalize(sent_uof['precinct'])
+        assert check_uof.shift == cleaner.capitalize(sent_uof['shift'])
+        assert check_uof.beat == cleaner.capitalize(sent_uof['beat'])
         assert check_uof.disposition == sent_uof['disposition']
-        assert check_uof.officer_force_type == Cleaners.officer_force_type(sent_uof['officerForceType'])
+        assert check_uof.officer_force_type == cleaner.officer_force_type(sent_uof['officerForceType'])
         assert check_uof.use_of_force_reason == sent_uof['useOfForceReason']
         assert check_uof.service_type == sent_uof['serviceType']
         assert check_uof.arrest_made == sent_uof['arrestMade']
@@ -384,15 +389,15 @@ class TestHeartbeat:
         assert check_uof.resident_hospitalized == sent_uof['residentHospitalized']
         assert check_uof.officer_injured == sent_uof['officerInjured']
         assert check_uof.officer_hospitalized == sent_uof['officerHospitalized']
-        assert check_uof.resident_race == Cleaners.race(sent_uof['residentRace'])
-        assert check_uof.resident_sex == Cleaners.sex(sent_uof['residentSex'])
-        assert check_uof.resident_age == Cleaners.number_to_string(sent_uof['residentAge'])
+        assert check_uof.resident_race == cleaner.race(sent_uof['residentRace'])
+        assert check_uof.resident_sex == cleaner.sex(sent_uof['residentSex'])
+        assert check_uof.resident_age == cleaner.number_to_string(sent_uof['residentAge'])
         assert check_uof.resident_condition == sent_uof['residentCondition']
         assert check_uof.officer_identifier == sent_uof['officerIdentifier']
-        assert check_uof.officer_race == Cleaners.race(sent_uof['officerRace'])
-        assert check_uof.officer_sex == Cleaners.sex(sent_uof['officerSex'])
-        assert check_uof.officer_age == Cleaners.number_to_string(sent_uof['officerAge'])
-        assert check_uof.officer_years_of_service == Cleaners.number_to_string(sent_uof['officerYearsOfService'])
+        assert check_uof.officer_race == cleaner.race(sent_uof['officerRace'])
+        assert check_uof.officer_sex == cleaner.sex(sent_uof['officerSex'])
+        assert check_uof.officer_age == cleaner.number_to_string(sent_uof['officerAge'])
+        assert check_uof.officer_years_of_service == cleaner.number_to_string(sent_uof['officerYearsOfService'])
         assert check_uof.officer_condition == sent_uof['officerCondition']
 
     def test_update_uof_data(self, testapp):
@@ -436,15 +441,16 @@ class TestHeartbeat:
         assert len(all_uof) == 1
 
         # check the uof incident in the database against the updated data that was sent
+        cleaner = Cleaners()
         sent_uof = updated_uof_data[0]
         check_uof = UseOfForceIncident.query.filter_by(opaque_id=sent_uof['opaqueId']).first()
         assert check_uof.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_uof['occuredDate']
-        assert check_uof.division == Cleaners.capitalize(sent_uof['division'])
-        assert check_uof.precinct == Cleaners.capitalize(sent_uof['precinct'])
-        assert check_uof.shift == Cleaners.capitalize(sent_uof['shift'])
-        assert check_uof.beat == Cleaners.capitalize(sent_uof['beat'])
+        assert check_uof.division == cleaner.capitalize(sent_uof['division'])
+        assert check_uof.precinct == cleaner.capitalize(sent_uof['precinct'])
+        assert check_uof.shift == cleaner.capitalize(sent_uof['shift'])
+        assert check_uof.beat == cleaner.capitalize(sent_uof['beat'])
         assert check_uof.disposition == sent_uof['disposition']
-        assert check_uof.officer_force_type == Cleaners.officer_force_type(sent_uof['officerForceType'])
+        assert check_uof.officer_force_type == cleaner.officer_force_type(sent_uof['officerForceType'])
         assert check_uof.use_of_force_reason == sent_uof['useOfForceReason']
         assert check_uof.service_type == sent_uof['serviceType']
         assert check_uof.arrest_made == sent_uof['arrestMade']
@@ -454,15 +460,15 @@ class TestHeartbeat:
         assert check_uof.resident_hospitalized == sent_uof['residentHospitalized']
         assert check_uof.officer_injured == sent_uof['officerInjured']
         assert check_uof.officer_hospitalized == sent_uof['officerHospitalized']
-        assert check_uof.resident_race == Cleaners.race(sent_uof['residentRace'])
-        assert check_uof.resident_sex == Cleaners.sex(sent_uof['residentSex'])
-        assert check_uof.resident_age == Cleaners.number_to_string(sent_uof['residentAge'])
+        assert check_uof.resident_race == cleaner.race(sent_uof['residentRace'])
+        assert check_uof.resident_sex == cleaner.sex(sent_uof['residentSex'])
+        assert check_uof.resident_age == cleaner.number_to_string(sent_uof['residentAge'])
         assert check_uof.resident_condition == sent_uof['residentCondition']
         assert check_uof.officer_identifier == sent_uof['officerIdentifier']
-        assert check_uof.officer_race == Cleaners.race(sent_uof['officerRace'])
-        assert check_uof.officer_sex == Cleaners.sex(sent_uof['officerSex'])
-        assert check_uof.officer_age == Cleaners.number_to_string(sent_uof['officerAge'])
-        assert check_uof.officer_years_of_service == Cleaners.number_to_string(sent_uof['officerYearsOfService'])
+        assert check_uof.officer_race == cleaner.race(sent_uof['officerRace'])
+        assert check_uof.officer_sex == cleaner.sex(sent_uof['officerSex'])
+        assert check_uof.officer_age == cleaner.number_to_string(sent_uof['officerAge'])
+        assert check_uof.officer_years_of_service == cleaner.number_to_string(sent_uof['officerYearsOfService'])
         assert check_uof.officer_condition == sent_uof['officerCondition']
 
     def test_post_uof_data_near_match_does_not_update(self, testapp):
@@ -525,25 +531,26 @@ class TestHeartbeat:
         assert response.json_body['added'] == ois_count
 
         # check the ois incident in the database against the data that was sent
+        cleaner = Cleaners()
         sent_ois = ois_data[0]
         check_ois = OfficerInvolvedShooting.query.filter_by(opaque_id=sent_ois['opaqueId']).first()
         assert check_ois.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_ois['occuredDate']
-        assert check_ois.division == Cleaners.capitalize(sent_ois['division'])
-        assert check_ois.precinct == Cleaners.capitalize(sent_ois['precinct'])
-        assert check_ois.shift == Cleaners.capitalize(sent_ois['shift'])
-        assert check_ois.beat == Cleaners.capitalize(sent_ois['beat'])
+        assert check_ois.division == cleaner.capitalize(sent_ois['division'])
+        assert check_ois.precinct == cleaner.capitalize(sent_ois['precinct'])
+        assert check_ois.shift == cleaner.capitalize(sent_ois['shift'])
+        assert check_ois.beat == cleaner.capitalize(sent_ois['beat'])
         assert check_ois.disposition == sent_ois['disposition']
-        assert check_ois.resident_race == Cleaners.race(sent_ois['residentRace'])
-        assert check_ois.resident_sex == Cleaners.sex(sent_ois['residentSex'])
-        assert check_ois.resident_age == Cleaners.number_to_string(sent_ois['residentAge'])
-        assert check_ois.resident_weapon_used == Cleaners.resident_weapon_used(sent_ois['residentWeaponUsed'])
+        assert check_ois.resident_race == cleaner.race(sent_ois['residentRace'])
+        assert check_ois.resident_sex == cleaner.sex(sent_ois['residentSex'])
+        assert check_ois.resident_age == cleaner.number_to_string(sent_ois['residentAge'])
+        assert check_ois.resident_weapon_used == cleaner.resident_weapon_used(sent_ois['residentWeaponUsed'])
         assert check_ois.resident_condition == sent_ois['residentCondition']
         assert check_ois.officer_identifier == sent_ois['officerIdentifier']
         assert check_ois.officer_weapon_used == sent_ois['officerForceType']
-        assert check_ois.officer_race == Cleaners.race(sent_ois['officerRace'])
-        assert check_ois.officer_sex == Cleaners.sex(sent_ois['officerSex'])
-        assert check_ois.officer_age == Cleaners.number_to_string(sent_ois['officerAge'])
-        assert check_ois.officer_years_of_service == Cleaners.string_to_integer(sent_ois['officerYearsOfService'])
+        assert check_ois.officer_race == cleaner.race(sent_ois['officerRace'])
+        assert check_ois.officer_sex == cleaner.sex(sent_ois['officerSex'])
+        assert check_ois.officer_age == cleaner.number_to_string(sent_ois['officerAge'])
+        assert check_ois.officer_years_of_service == cleaner.string_to_integer(sent_ois['officerYearsOfService'])
         assert check_ois.officer_condition == sent_ois['officerCondition']
 
     def test_post_mistyped_ois_data(self, testapp):
@@ -576,25 +583,26 @@ class TestHeartbeat:
         assert response.json_body['added'] == ois_count
 
         # check the ois incident in the database against the data that was sent
+        cleaner = Cleaners()
         sent_ois = ois_data[0]
         check_ois = OfficerInvolvedShooting.query.filter_by(opaque_id=sent_ois['opaqueId']).first()
         assert check_ois.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_ois['occuredDate']
-        assert check_ois.division == Cleaners.capitalize(sent_ois['division'])
-        assert check_ois.precinct == Cleaners.capitalize(sent_ois['precinct'])
-        assert check_ois.shift == Cleaners.capitalize(sent_ois['shift'])
-        assert check_ois.beat == Cleaners.capitalize(sent_ois['beat'])
+        assert check_ois.division == cleaner.capitalize(sent_ois['division'])
+        assert check_ois.precinct == cleaner.capitalize(sent_ois['precinct'])
+        assert check_ois.shift == cleaner.capitalize(sent_ois['shift'])
+        assert check_ois.beat == cleaner.capitalize(sent_ois['beat'])
         assert check_ois.disposition == sent_ois['disposition']
-        assert check_ois.resident_race == Cleaners.race(sent_ois['residentRace'])
-        assert check_ois.resident_sex == Cleaners.sex(sent_ois['residentSex'])
-        assert check_ois.resident_age == Cleaners.number_to_string(sent_ois['residentAge'])
-        assert check_ois.resident_weapon_used == Cleaners.resident_weapon_used(sent_ois['residentWeaponUsed'])
+        assert check_ois.resident_race == cleaner.race(sent_ois['residentRace'])
+        assert check_ois.resident_sex == cleaner.sex(sent_ois['residentSex'])
+        assert check_ois.resident_age == cleaner.number_to_string(sent_ois['residentAge'])
+        assert check_ois.resident_weapon_used == cleaner.resident_weapon_used(sent_ois['residentWeaponUsed'])
         assert check_ois.resident_condition == sent_ois['residentCondition']
         assert check_ois.officer_identifier == sent_ois['officerIdentifier']
         assert check_ois.officer_weapon_used == sent_ois['officerForceType']
-        assert check_ois.officer_race == Cleaners.race(sent_ois['officerRace'])
-        assert check_ois.officer_sex == Cleaners.sex(sent_ois['officerSex'])
-        assert check_ois.officer_age == Cleaners.number_to_string(sent_ois['officerAge'])
-        assert check_ois.officer_years_of_service == Cleaners.string_to_integer(sent_ois['officerYearsOfService'])
+        assert check_ois.officer_race == cleaner.race(sent_ois['officerRace'])
+        assert check_ois.officer_sex == cleaner.sex(sent_ois['officerSex'])
+        assert check_ois.officer_age == cleaner.number_to_string(sent_ois['officerAge'])
+        assert check_ois.officer_years_of_service == cleaner.string_to_integer(sent_ois['officerYearsOfService'])
         assert check_ois.officer_condition == sent_ois['officerCondition']
 
     def test_update_ois_data(self, testapp):
@@ -637,25 +645,26 @@ class TestHeartbeat:
         assert len(all_ois) == 1
 
         # check the ois incident in the database against the updated data that was sent
+        cleaner = Cleaners()
         sent_ois = updated_ois_data[0]
         check_ois = OfficerInvolvedShooting.query.filter_by(opaque_id=sent_ois['opaqueId']).first()
         assert check_ois.occured_date.strftime('%Y-%m-%d %-H:%-M:%S') == sent_ois['occuredDate']
-        assert check_ois.division == Cleaners.capitalize(sent_ois['division'])
-        assert check_ois.precinct == Cleaners.capitalize(sent_ois['precinct'])
-        assert check_ois.shift == Cleaners.capitalize(sent_ois['shift'])
-        assert check_ois.beat == Cleaners.capitalize(sent_ois['beat'])
+        assert check_ois.division == cleaner.capitalize(sent_ois['division'])
+        assert check_ois.precinct == cleaner.capitalize(sent_ois['precinct'])
+        assert check_ois.shift == cleaner.capitalize(sent_ois['shift'])
+        assert check_ois.beat == cleaner.capitalize(sent_ois['beat'])
         assert check_ois.disposition == sent_ois['disposition']
-        assert check_ois.resident_race == Cleaners.race(sent_ois['residentRace'])
-        assert check_ois.resident_sex == Cleaners.sex(sent_ois['residentSex'])
-        assert check_ois.resident_age == Cleaners.number_to_string(sent_ois['residentAge'])
-        assert check_ois.resident_weapon_used == Cleaners.resident_weapon_used(sent_ois['residentWeaponUsed'])
+        assert check_ois.resident_race == cleaner.race(sent_ois['residentRace'])
+        assert check_ois.resident_sex == cleaner.sex(sent_ois['residentSex'])
+        assert check_ois.resident_age == cleaner.number_to_string(sent_ois['residentAge'])
+        assert check_ois.resident_weapon_used == cleaner.resident_weapon_used(sent_ois['residentWeaponUsed'])
         assert check_ois.resident_condition == sent_ois['residentCondition']
         assert check_ois.officer_identifier == sent_ois['officerIdentifier']
         assert check_ois.officer_weapon_used == sent_ois['officerForceType']
-        assert check_ois.officer_race == Cleaners.race(sent_ois['officerRace'])
-        assert check_ois.officer_sex == Cleaners.sex(sent_ois['officerSex'])
-        assert check_ois.officer_age == Cleaners.number_to_string(sent_ois['officerAge'])
-        assert check_ois.officer_years_of_service == Cleaners.string_to_integer(sent_ois['officerYearsOfService'])
+        assert check_ois.officer_race == cleaner.race(sent_ois['officerRace'])
+        assert check_ois.officer_sex == cleaner.sex(sent_ois['officerSex'])
+        assert check_ois.officer_age == cleaner.number_to_string(sent_ois['officerAge'])
+        assert check_ois.officer_years_of_service == cleaner.string_to_integer(sent_ois['officerYearsOfService'])
         assert check_ois.officer_condition == sent_ois['officerCondition']
 
     def test_post_ois_data_near_match_does_not_update(self, testapp):

